src/features/filter.ts

```diff
@@ -1,4 +1,10 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Status } from '../types/Status';
+
+export interface TypeFilter {
+  query: string;
+  status: string;
```

Comment:
The `status` property in `TypeFilter` should be of type `Status` instead of `string` to ensure type safety and consistency with the `setStatus` reducer.